src/main/privacy/Controller/PrivacyController.php

```diff
@@ -2,55 +2,70 @@
 
 namespace Claroline\PrivacyBundle\Controller;
 
-use Claroline\AppBundle\API\Utils\ArrayUtils;
-use Claroline\AppBundle\Controller\AbstractSecurityController;
+use Claroline\AppBundle\API\Crud;
+use Claroline\AppBundle\API\SerializerProvider;
 use Claroline\AppBundle\Controller\RequestDecoderTrait;
-use Claroline\CoreBundle\API\Serializer\ParametersSerializer;
+use Claroline\AppBundle\Persistence\ObjectManager;
 use Claroline\CoreBundle\Library\Configuration\PlatformConfigurationHandler;
+use Claroline\CoreBundle\Validator\Exception\InvalidDataException;
+use Claroline\PrivacyBundle\Entity\Privacy;
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
 
-class PrivacyController extends AbstractSecurityController
+class PrivacyController
```

Comment:
Il aurait fallu garder le `AbstractSecurityController` pour pouvoir sécuriser ton endpoint avec `$this->canOpenAdminTool('privacy');`